Description: Add note to dnswalk.readme about dnswalk being available as a Debian package.
Forwarded: not-needed
Author: Florian Hinzmann <fh@debian.org>
Index: libnet-dns-perl/contrib/dnswalk.README
===================================================================
--- libnet-dns-perl.orig/contrib/dnswalk.README	2009-02-28 19:07:52.000000000 +0100
+++ libnet-dns-perl/contrib/dnswalk.README	2010-03-15 10:24:42.000000000 +0100
@@ -4,3 +4,8 @@
 
   http://sourceforge.net/projects/dnswalk/
 
+
+NOTE for Debian users:
+  As dnswalk is packaged for Debian you may as well install the
+  Debian package (aptitude install dnswalk).
+    -- Florian Hinzmann <fh@debian.org>
